functions/Get-DbaAgentSchedule.ps1

```diff
@@ -0,0 +1,78 @@
+FUNCTION Get-DbaAgentSchedule
```

Comment:
@potatoqualitee @DevOpsFu @wsmelton 
There are 2 different type of schedules in SQLServer:
jobserver.sharedschedule
jobserver.job.jobschedule
Given there are also other schedules outside, it's important to clearly name them.
As @wsmelton  says, `Get-DbaAgentSharedSchedule` for this one (if the New- also works on this type, indeed that needs to be `New-DbaAgentSharedSchedule`, @DevOpsFu )
Another function should be named `Get-DbaAgentJobSchedule`, isn't Sander working on that?